README.md

```diff
@@ -0,0 +1,55 @@
+`tf_lambda_s3_backup`
+---------------------
+
+This modules allows configure a ECS task with a given container to be triggered using
+a cron expression.
+
+Currently is designed to trigger backups using the container: https://github.com/mergermarket/docker-mysql-s3-backup,
+but it would be adapted to run any container.
+
+*Note*: since the release of [scheduled ECS tasks from AWS](https://aws.amazon.com/about-aws/whats-new/2017/06/amazon-ecs-now-supports-time-and-event-based-task-scheduling), there is no point to use a lambda for this anymore. But the code related to tasks can be reused.
```

Comment:
Yes it is worth it indeed. I did the PR basically because I had it finished and we can one day benefit from some of the code here. 

I think the scheduled ECS tasks will be far easier and simpler, but need to check if we can configure that with terraform or if the feature needs to be patched. 

I wanted to check that but I did not have time :(